src/modules/mavlink/mavlink_params.c

```diff
@@ -175,3 +175,16 @@ PARAM_DEFINE_INT32(MAV_HB_FORW_EN, 1);
  * @group MAVLink
  */
 PARAM_DEFINE_INT32(MAV_ODOM_LP, 0);
+
+/**
+ * Timeout in seconds for the RADIO_STATUS reports coming in
+ *
+ * If the connected radio stops reporting RADIO_STATUS for a certain time,
+ * a warning is triggered and, if MAV_X_RADIO_CTL is enabled, the software-flow
+ * control is reset.
+ *
+ * @group MAVLink
+ * @min 1
+ * @max 250
```

Comment:
Can you specify the unit?

Reply:
Again? Sure, I didn't know that's required

Reply:
So apparently, no:
![image](https://user-images.githubusercontent.com/14265408/74329061-5e8a3780-4d8f-11ea-9237-37e54da0bd88.png)

Reply:
You should use the @unit tag

Reply:
gotcha, thanks